tools/prereqs/setup_azure_cli.sh

```diff
@@ -9,12 +9,13 @@ require_root
 function apt_install() {
     # prompt for confirmation
     echo "This script will install the latest version of the Azure CLI using the Microsoft APT repo."
-    read -p "Do you wish to continue? " -n 1 -r
-    echo
-    if [[ ! $REPLY =~ ^[Yy]$ ]]
-    then
-        exit 1
-    fi
+    # next 6 lines commented due to automation
```

Comment:
All these changes to the shell scripts make it difficult to re-use as they previously were. I suggest adding an override instead of commenting out these blocks.

Reply:
Setting an environment variable in the Dockerfile like `ASSUME_YES` or `NO_INPUT`, maybe prefixed with `BEDROCK_` could work?